src/main/java/com/gluonhq/substrate/target/AbstractTargetConfiguration.java

```diff
@@ -405,11 +411,23 @@ private Path createJNIConfig(String suffix) throws IOException {
                     bw.write(line + "\n");
                 }
             }
+            for (String javaClass : projectConfiguration.getJniList()) {
+                writeEntry(bw, javaClass);
+            }
             bw.write("]");
         }
         return jniPath;
     }
 
+    private static void writeEntry(BufferedWriter bw, String javaClass) throws IOException {
+        writeEntry(bw, javaClass, false);
+    }
+
+    private static void writeEntry(BufferedWriter bw, String javaClass, boolean exclude) throws IOException {
```

Comment:
we need better names or docs. e.g. what is "exclude"?

Reply:
The `exclude` parameter might not be used at all, since we are adding concise methods now in the jni/reflection files. Since this is still work in progress, we'll have probably to refactor it later on anyway.